core/metrics-core-service/src/main/java/org/hawkular/metrics/core/jobs/JobsServiceImpl.java

```diff
@@ -71,7 +73,7 @@ public void shutdown() {
     @Override
     public Single<JobDetails> submitDeleteTenantJob(String tenantId) {
         return scheduler.scheduleJob(DeleteTenant.JOB_NAME, DeleteTenant.JOB_NAME, ImmutableMap.of("tenantId",
-                tenantId), new SingleExecutionTrigger.Builder().build());
+                tenantId), new SingleExecutionTrigger.Builder().withDelay(5, TimeUnit.MINUTES).build());
```

Comment:
Why a 5 minutes delay? What happens if the user resends the request to delete a tenant?